horovod/run/elastic/worker.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2020 Uber Technologies, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from __future__ import absolute_import
+
+import os
+import threading
+
+from horovod.run.common.util import network, secret
+from horovod.run.elastic.rendezvous import PUT_WORKER_ADDRESSES
+from horovod.run.http.http_client import put_data_into_kvstore
+
+
+HOROVOD_GLOO_RENDEZVOUS_ADDR = 'HOROVOD_GLOO_RENDEZVOUS_ADDR'
+HOROVOD_GLOO_RENDEZVOUS_PORT = 'HOROVOD_GLOO_RENDEZVOUS_PORT'
+HOROVOD_GLOO_IFACE = 'HOROVOD_GLOO_IFACE'
+HOROVOD_HOSTNAME = 'HOROVOD_HOSTNAME'
+HOROVOD_LOCAL_RANK = 'HOROVOD_LOCAL_RANK'
+
+
+class HostsUpdatedRequest(object):
+    """Notifies worker that the set of available hosts/slots has changed."""
+    def __init__(self, timestamp):
+        self.timestamp = timestamp
+
+
+class HostsUpdatedResponse(object):
```

Comment:
empty response, why not using `network.AckResponse`?

Reply:
Good point.  Holdover from an earlier implementation.